sdk/storage_blobs/src/blob/operations/get_blob.rs

```diff
@@ -58,12 +38,12 @@ impl<'a> GetBlobBuilder<'a> {
     setters! {
         range: Range => Some(range),
         blob_versioning: &'a BlobVersioning => Some(blob_versioning),
-        client_request_id: ClientRequestId<'a> => Some(client_request_id),
+        client_request_id: &'a ClientRequestId => Some(client_request_id),
         timeout: Timeout => Some(timeout),
-        lease_id: &'a LeaseId => Some(lease_id),
+        lease_id: LeaseId => Some(lease_id),
     }
 
-    pub async fn execute(&self) -> azure_core::Result<GetBlobResponse> {
+    async fn execute(&self) -> azure_core::Result<GetBlobResponse> {
```

Comment:
Is there a reason this wasn't moved to using the pipeline architecture?

Reply:
Note, this didn't move everything to the pipeline architecture, just into_future.  There is more work to be done to enable that, primarily exposing a Context.

Note, this is still a hold over because Pageable isn't used yet, as this doesn't use pipelines yet.